scripts/create-package/plop-templates-react/package.json.hbs

```diff
@@ -44,9 +46,11 @@
     "react-test-renderer": ""
   },
   "dependencies": {
-    "@fluentui/theme": "",
-    "@fluentui/react-compose": "",
+    "@fluentui/react-hooks": "",
+    "@fluentui/react-make-styles": "",
+    "@fluentui/react-theme": "",
     "@fluentui/react-theme-provider": "",
```

Comment:
Can you please check, that this is necessary dependency?

Reply:
Apparently `@fluentui/react-theme` is useless!